src/components/Charts/ModelChart.js

```diff
@@ -26,116 +30,140 @@ const ModelChart = ({
   interventions,
   currentIntervention,
 }) => {
-  const { isEmbed } = useEmbed();
-  const interventionToModel = {
-    [INTERVENTIONS.LIMITED_ACTION]: interventions.baseline,
-    [INTERVENTIONS.SOCIAL_DISTANCING]:
-      interventions.distancingPoorEnforcement.now,
-    [INTERVENTIONS.SHELTER_IN_PLACE]: interventions.distancing.now,
-  };
-
-  let model = interventionToModel[currentIntervention];
+  let interventionsModelIsValid = true;
+  let model;
 
-  if (currentIntervention === INTERVENTIONS.SHELTER_IN_PLACE) {
-    model = interventionToModel[INTERVENTIONS.SOCIAL_DISTANCING];
+  // In future we can do further model shape/schema checking
+  try {
+    let interventionToModel = interventionToModelMap(interventions);
+    let worstCaseIntervention = getWorstCaseIntervention(currentIntervention);
+    model = interventionToModel[worstCaseIntervention];
+  } catch (e) {
+    interventionsModelIsValid = false;
```

Comment:
This just adds some error handling for bad model data, which was an issue I kept running into when I was testing. This should help it fail gracefully.

The changes below look more significant than they are, really they're just only evaluating all of these variables if the model data is valid to avoid everything breaking when it isn't.